client/components/chart/style.scss

```diff
@@ -309,6 +309,7 @@
 				}
 
 				.woocommerce-legend__item-checkmark {
+					border: 1px solid $core-grey-light-700;
```

Comment:
I think this should be `$core-grey-light-900` - https://automattic.invisionapp.com/d/main#/console/14672444/310038618/inspect